sct.py

```diff
@@ -1372,13 +1373,24 @@ def create_runner_instance(cloud_provider, region, availability_zone, instance_t
         sys.exit(1)
 
 
+@cli.command("set-runner-tags")
+@click.argument("runner-ip", type=str)
+@click.option("-t", "--tags", type=(str, str),
+              help="Space separated key value pair to add as a new tag to the runner",
+              multiple=True)
+def set_runner_tags(runner_ip, tags):
+    add_file_logger()
+    update_sct_runner_tags(test_runner_ip=runner_ip, tags=dict(tags))
+
+
 @cli.command("clean-runner-instances", help="Clean all unused SCT runner instances")
 @click.option("-ts", "--test-status", required=False, type=str, default="FAILED")
 @click.option("-ip", "--runner-ip", required=False, type=str, default="")
 @click.option('--dry-run', is_flag=True, default=False, help='dry run')
-def clean_runner_instances(test_status, runner_ip, dry_run):
+@click.option("--logs-collected", default=False, help='Were the logs collected')
```

Comment:
in commands bash like, we usually don't send `True` or `False`, but add the parameter (or not)...
so it would most likely have `is_flag=True` in it, so no values must be passed in addition to the parameter itself..
so the code that will make this call in the `groovy` file, will check `if logs collected; then cmd += --logs-collected`

Reply:
@fgelcer I changed the name of this argument to "collect_logs_result", since that's the name of the stage that we're interested in here. I'm hesitant to treat this as a flag, since it's a piece of data we're passing in to the function, not a switch for the function's behavior (in contrast to `--dry-run` here.

@fruch It is a boolean, but as mentioned before, it's not something that we enable / disable, it's data about a different stage's success or failure that we're passing in here. I can add an explanation for how this data is treated, but it would quite verbose for a CLI help text snippet, since it will modify the SCT runners tags based on logic involving also checking the runner's log etc.

Reply:
maybe it would be clearer if it was called `were_logs_collected` or something like that?